static/beta/prod/navigation/rhel-navigation.json

```diff
@@ -214,6 +214,14 @@
         {
           "title": "Compliance",
           "expandable": true,
+          "alt_title": [
```

Comment:
I don't think this one is used anywhere in the services.json right? It won't be uploaded to the search index.